conf/messages.cy

```diff
@@ -469,32 +469,63 @@ pdf.ruling.aboutDecision.text=Rwyf wedi gwneud y penderfyniad hwn o dan adran 24
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements=Dylai'r cais hwn cynnwys copïau o'r canlynol:
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements.1=y llythyr hwn, neu'r llythyr canlyniad yr adolygiad
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements.2=unrhyw asesiad
+pdf.ruling.section.appealForIndependentTribunal.relevantInfo=Os nad ydych yn anfon yr wybodaeth berthnasol, gall y Gwasanaeth Llysoedd a Thribiwnlysoedd EF wrthod eich apêl.
+pdf.ruling.section.appealForIndependentTribunal.tribunalsInfo=I gael rhagor o wybodaeth am dribiwnlysoedd ewch i'r dudalen isod neu ffoniwch 0300 303 5176.
+pdf.ruling.section.appealForIndependentTribunal.page=<a href="www.gov.uk/tribiwnlys-treth" target="_blank">www.gov.uk/tribiwnlys-treth</a>
+
+pdf.ruling.section.appealsAndReviews.title=Rhagor o wybodaeth am apeliadau ac adolygiadau
+pdf.ruling.section.appealsAndReviews.moreInfo=I gael rhagor o wybodaeth am eich hawliau o ran apelio ac adolygiadau, ewch i'r dudalen ganlynol:
+pdf.ruling.section.appealsAndReviews.page=<a href="www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision.cy" target="_blank">www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision.cy</a>
+pdf.ruling.section.appealsAndReviews.copy=Gallwch ofyn am gopi o'r ddogfen HMRC1 drwy ffonio 0300 200 1900.
+
+pdf.ruling.section.communicatingByEmail.title=Gwybodaeth bwysig am ohebu drwy e-bost
+pdf.ruling.section.communicatingByEmail.tellUs=Os hoffech ohebu drwy e-bost, mae'n rhaid i chi roi gwybod i ni eich bod yn deall ac yn derbyn y risgiau o wneud hynny. Manylir ar y rhain yn ein taflen wybodaeth CC/FS72 DSC1, 'Gohebu â CThEF drwy e-bost'. I ddod o hyd i'r daflen wybodaeth hon, ewch i'r dudalen:
+pdf.ruling.section.communicatingByEmail.page=<a href="www.gov.uk/government/publications/corresponding-with-hmrc-by-email-ccfs72.cy" target="_blank">www.gov.uk/government/publications/corresponding-with-hmrc-by-email-ccfs72.cy</a>
+pdf.ruling.section.communicatingByEmail.copy=Gallwch ofyn am gopi o daflen wybodaeth DSC1 drwy ffonio 0300 200 1900.
+
+pdf.ruling.section.acknowledgement.title=Os ydych yn deall ac yn derbyn y risgiau o ddefnyddio e-bost, dylech gynnwys y gydnabyddiaeth hon o fewn unrhyw ohebiaeth gychwynnol.
+pdf.ruling.section.acknowledgement=Ni fyddwn yn cysylltu â chi drwy e-bost ynghylch mater sy'n ymwneud â threth, oni bai eich bod eisoes wedi rhoi caniatâd i ni wneud hynny. Os oes amheuaeth gennych ynghylch dilysrwydd e-bost sy'n honni ei fod yn dod oddi wrth CThEF, peidiwch â chlicio ar unrhyw gysylltiadau, rhoi unrhyw fanylion personol nac ateb yr e-bost chwaith. Anfonwch yr e-bost ymlaen atom yn: <a class="govuk-link" href= "mailto:gwasanaeth.cymraeg@hmrc.gov.uk">gwasanaeth.cymraeg@hmrc.gov.uk</a>
```

Comment:
Apostrophes are not displayed currently. Please use the curved one for consistency in welsh. Update to `sy’n` (in two places) thanks